tests/test_npm_meta.py

```diff
@@ -3,17 +3,16 @@
 import shutil
 import marshmallow_dataclass
 import mrrc.archive as archive
-import mrrc.metadata_version_npm as npm_version
-import mrrc.metadata_package_npm as npm_package
-from mrrc.metadata_package_npm import NPMPackageMetadata
+import mrrc.metadata_npm as metadata_npm
+from mrrc.metadata_npm import NPMPackageMetadata
```

Comment:
As python function can be import here, so I think you can directly use:
from mrrc.metadata_npm import NPMPackageMetadata, scan_for_version, gen_package_metadata_file

and just use the functions directly in your code below

Reply:
Okay good, thank you